x-pack/platform/plugins/shared/stack_connectors/server/connector_types/openai/openai.ts

```diff
@@ -53,18 +57,33 @@ import {
   getRequestWithStreamOption,
   pipeStreamingResponse,
   sanitizeRequest,
+  validatePKICertificates,
+  getPKISSLOverrides,
+  formatPEMContent,
 } from './lib/utils';
 
 export class OpenAIConnector extends SubActionConnector<Config, Secrets> {
-  private url;
-  private provider;
-  private key;
-  private openAI;
-  private headers;
+  private url: string;
+  private provider: OpenAiProviderType;
+  private key: string;
+  private openAI: OpenAI;
+  private headers: Record<string, string>;
+  private configAny: Config & {
```

Comment:
please get rid of `configAny`. Since these properties are part of `Config`, you can check for them with `'certificateData' in this.config` before using them to avoid type errors